profile.h

```diff
@@ -103,6 +103,7 @@ enum bcm2_arch
 	BCM2_3383 = 0x3383,
 	BCM2_3384 = 0x3384,
 	BCM2_3390 = 0x3390,
+	BCM2_3843 = 0x3843,
```

Comment:
Please keep this sorted (i.e. put it before `BCM2_3345`)

Reply:
I just noticed it's `3843`, not `3343`. However, all Broadcom CM chipsets start with `33`. According to [deviwiki](https://deviwiki.com/wiki/Netgear_CM500) this should be `33843`!